xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -1473,5 +1473,55 @@ test_build:
         </listitem>
       </itemizedlist>
     </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.use_cases.config_url_and_placeholder_variables">
```

Comment:
@hellcp-work maybe you can also reference to this new chapter from "Placeholder Variables" chapter. So when someone reads about then immediately can see and find what is possible to do with it